pages/getcertified.html

```diff
@@ -4,26 +4,31 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link
       href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,700;1,300;1,400&display=swap"
-      rel="stylesheet"
-    />
+      rel="stylesheet" />
     <link href="../CSS/getcertified.css" rel="stylesheet" />
     <link href="../CSS/form.css" rel="stylesheet" />
     <link href="../CSS/footer.css" rel="stylesheet" />
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/emailjs-com@2/dist/email.min.js"></script>
```

Comment:
Lets stick all your scripts at the bottom -- right before the closing `</body>` tag. Why do this? Well the browser reads from top to bottom. This way all our html and css is loaded before our javascript, meaning the user gets to see the page faster even if its not quite fully usable yet. To be honest I say this but its getting to be an outdated technique but for now is still good practice. The reason its getting outdated is browsers and the internet have upgraded like beyonce. HTTP2, what we think of as the modern web is about to become HTTP3, in fact parts of it already are. And its a lot faster and handles data a bit differently and much faster. All that to say bullshit tricks like this where you put javascript right before the closing body tag are about to be a thing of the past. If you wanna blow your brain up with HTTP3 this isnt a bad place to start: https://medium.com/ably-realtime/http-3-deep-dive-9318f7d6834d